services/QuillLMS/app/mailers/stripe_integration/mailer.rb

```diff
@@ -8,7 +8,7 @@ class Mailer < ::ApplicationMailer
     CHARGE_DISPUTE_CREATED_URL = "#{STRIPE_DASHBOARD_URL}/disputes?statuses[0]=needs_response"
 
     def charge_dispute_created
-      mail(to: QUILL_TEAM_EMAIL_ADDRESS, subject: 'Charge Dispute Created') do |format|
+      mail(to: ENV.fetch('STRIPE_NOTIFICATIONS_EMAIL'), subject: 'Charge Dispute Created') do |format|
```

Comment:
This isn't a huge deal, but I think we've been moving to sort of proxy `ENV` variables as class constants.  I don't know that it's better, but consistency is good, so how about we just create a `STRIPE_NOTIFICATION_EMAIL = ENV.fetch('STRIPE_NOTIFICATION_EMAIL')` constant.  Then the spec can stub the class constant rather than ENV.

Reply:
Good point.  I will update.